apps/tailwind-components/components/input/RadioIcon.vue

```diff
@@ -10,13 +10,17 @@
       cx="10"
       cy="10"
       r="9"
-      stroke-width="1"
+      :style="{ 'stroke-width': invalid ? 2 : 1 }"
```

Comment:
I'm not sure if single radio input needs to be styled if it is invalid. Modifying the stroke width will cause content shifting. It is more likely radio buttons will always be part of the radio group component (i.e., a fieldset). The styling should be moved to the group.